[19 01 2017 09:58:08.259] [DEBUG] - Mongoose is now disconnected..!
[19 01 2017 09:58:08.261] [ERROR] - Error in Mongoose connection:  { MongoError: failed to connect to server [127.0.0.1:27017] on first connect
    at Pool.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\topologies\server.js:326:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\pool.js:272:12)
    at Connection.g (events.js:291:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\connection.js:175:49)
    at Socket.g (events.js:291:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1278:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
  name: 'MongoError',
  message: 'failed to connect to server [127.0.0.1:27017] on first connect' }
[19 01 2017 10:00:33.212] [DEBUG] - Mongoose is now disconnected..!
[19 01 2017 10:00:33.215] [ERROR] - Error in Mongoose connection:  { MongoError: failed to connect to server [127.0.0.1:27017] on first connect
    at Pool.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\topologies\server.js:326:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\pool.js:272:12)
    at Connection.g (events.js:291:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\connection.js:175:49)
    at Socket.g (events.js:291:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1278:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
  name: 'MongoError',
  message: 'failed to connect to server [127.0.0.1:27017] on first connect' }
[19 01 2017 10:02:44.314] [DEBUG] - Mongoose is now disconnected..!
[19 01 2017 10:02:44.315] [ERROR] - Error in Mongoose connection:  { MongoError: failed to connect to server [127.0.0.1:27017] on first connect
    at Pool.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\topologies\server.js:326:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\pool.js:272:12)
    at Connection.g (events.js:291:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\connection.js:175:49)
    at Socket.g (events.js:291:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1278:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
  name: 'MongoError',
  message: 'failed to connect to server [127.0.0.1:27017] on first connect' }
[19 01 2017 10:04:39.005] [DEBUG] - Mongoose is now disconnected..!
[19 01 2017 10:04:39.007] [ERROR] - Error in Mongoose connection:  { MongoError: failed to connect to server [127.0.0.1:27017] on first connect
    at Pool.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\topologies\server.js:326:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\pool.js:272:12)
    at Connection.g (events.js:291:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\connection.js:175:49)
    at Socket.g (events.js:291:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1278:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
  name: 'MongoError',
  message: 'failed to connect to server [127.0.0.1:27017] on first connect' }
[19 01 2017 10:12:42.318] [DEBUG] - Mongoose is now disconnected..!
[19 01 2017 10:12:42.323] [ERROR] - Error in Mongoose connection:  { MongoError: failed to connect to server [127.0.0.1:27017] on first connect
    at Pool.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\topologies\server.js:326:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\pool.js:272:12)
    at Connection.g (events.js:291:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\connection.js:175:49)
    at Socket.g (events.js:291:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1278:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
  name: 'MongoError',
  message: 'failed to connect to server [127.0.0.1:27017] on first connect' }
[19 01 2017 10:13:07.201] [DEBUG] - Mongoose is now disconnected..!
[19 01 2017 10:13:07.204] [ERROR] - Error in Mongoose connection:  { MongoError: failed to connect to server [127.0.0.1:27017] on first connect
    at Pool.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\topologies\server.js:326:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\pool.js:272:12)
    at Connection.g (events.js:291:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\connection.js:175:49)
    at Socket.g (events.js:291:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1278:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
  name: 'MongoError',
  message: 'failed to connect to server [127.0.0.1:27017] on first connect' }
[19 01 2017 10:13:18.690] [DEBUG] - Mongoose is now disconnected..!
[19 01 2017 10:13:18.692] [ERROR] - Error in Mongoose connection:  { MongoError: failed to connect to server [127.0.0.1:27017] on first connect
    at Pool.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\topologies\server.js:326:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\pool.js:272:12)
    at Connection.g (events.js:291:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\connection.js:175:49)
    at Socket.g (events.js:291:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1278:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
  name: 'MongoError',
  message: 'failed to connect to server [127.0.0.1:27017] on first connect' }
[19 01 2017 10:14:37.370] [DEBUG] - Mongoose is now connected to  mongodb://127.0.0.1:27017/zynla
[19 01 2017 10:15:12.302] [DEBUG] - Mongoose is now connected to  mongodb://127.0.0.1:27017/zynla
[19 01 2017 10:17:02.837] [DEBUG] - Mongoose is now connected to  mongodb://127.0.0.1:27017/zynla
[19 01 2017 10:17:25.135] [DEBUG] - Mongoose is now connected to  mongodb://127.0.0.1:27017/zynla
[19 01 2017 10:17:48.059] [DEBUG] - Mongoose is now connected to  mongodb://127.0.0.1:27017/zynla
[19 01 2017 10:17:58.756] [DEBUG] - Mongoose is now connected to  mongodb://127.0.0.1:27017/zynla
